demos/svg/lv_demo_svg.c

```diff
@@ -0,0 +1,80 @@
+/**
+ * @file lv_demo_svg.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "lv_demo_svg.h"
+
+#if LV_USE_DEMO_SVG
```

Comment:
I meant this demo is not required. We usually put mode complex UIs into the demos folder. It's enough to keep it as an example.

Reply:
done